File: src/Entities/Medicine.Entities/Models/CourseSettings.cs
using Medicine.Entities.Enums;
using Medicine.Entities.Models.Base;

namespace Medicine.Entities.Models
{
    public class CourseSettings : EntityAuthor
    {
        public int  CourseId  { get; set; }
        public Course Course  { get; set; }
        public Sex Sex { get; set; }
        public int? MinAge { get; set; }
        public int? MaxAge { get; set; }
        public int Weight { get; set; }

    }
}

File: front/medicine/src/app/shared/models/entity/CourseSettings.ts
import { Sex } from "../../enums/Sex";
import { User } from "./User";
import { Course } from "./Course";

export class CourseSettings {
  courseId!: number;
  course!: Course;
  sex!: Sex;
  minAge?: number;
  maxAge?: number;
  weight!: number;
  user?: User;
  userId?: number;
  id!: number;
  createdAt?: Date;
}

